src/app/_layout.tsx

```diff
@@ -1,10 +1,15 @@
 import { Stack } from 'expo-router';
 import React from 'react';
 
+import { AuthContextProvider } from '../context/AuthContext';
+
 export default function StackLayout() {
   return (
-    <Stack screenOptions={{ headerShown: false }}>
-      <Stack.Screen name="(BottomTabNavigation)" options={{}} />
-    </Stack>
+    <AuthContextProvider>
+      <Stack screenOptions={{ headerShown: false }}>
+        <Stack.Screen name="(Authentication)" options={{}} />
```

Comment:
I added the `(Authentication)` folder within the AuthContext so it can also utilize the auth functions.